go.mod

```diff
@@ -6,15 +6,15 @@ replace k8s.io/client-go => k8s.io/client-go v0.20.0
 
 require (
 	github.com/evanphx/json-patch v4.9.0+incompatible
-	github.com/go-logr/logr v0.3.0
-	github.com/rancher/eks-operator v1.0.6-rc1
+	github.com/go-logr/logr v0.3.0 // indirect
+	github.com/rancher/eks-operator v1.0.6
```

Comment:
We need to look into this and whats going on, but merging this PR first to unblock QA